examples/run_largecyc_generic_echo.py

```diff
@@ -136,7 +134,7 @@
     [len(ph1), len(ph_overall), -1],
 )
 data.setaxis("ph1", ph1 / 4).setaxis("ph_overall", ph_overall / 4)
-data.set_prop("postproc_type", "spincore_generalproc_v1")
+data.set_prop("postproc_type", "spincore_generalproc_v3")
```

Comment:
This should be named after the ppg. It should not be called generalproc (this was a mistake I didn't catch before)